### ISSUE TITLE
ember-cli-less 1.5.6 does not find function less.writeError

### ISSUE THREAD

**Issue by iqdoq-dfischer** [gpoitch/ember-cli-less#64]
I am using ember-cli 2.18.2 on a windows x64 pc with node v8.9.3.

In package.json using ember-cli-less ^1.5.5 installed the new ember-cli-less 1.5.6 today.
With version 1.5.6 I get the following error while running ember build:

```
[...]
cleaning up...
Build failed.
Build Error (LessCompiler)

less.writeError is not a function
```
.. and the build fails with exit code 1.

If I downgrade ember-cli-less to version 1.5.5 everything is fine.

I now also pinned broccoli-less-single to ^0.6.4 because broccoli-less-single 1.0.0 installs less v3.x instead of the former v2.x used by broccoli-less-single v<1.0.

**Comment by knownasilya** [comment]
cc @jasonmit

**Comment by jasonmit** [comment]
@knownasilya looking into it now.  Thanks

**Comment by jasonmit** [comment]
@knownasilya patched in 1.0.1 https://github.com/gabrielgrant/broccoli-less-single/commit/b90e2eb7e9101c51b401484ee20bfbf2bd9201b7

**Comment by knownasilya** [comment]
Released v1.5.7 to revert to Less v2. Releasing ember-cli-less v2 for Less v3 now.

**Comment by iqdoq-dfischer** [comment]
I can confirm that my issues have been solved with version 1.5.7 of ember-cli-less.
Thank you very much!
:thumbsup:
